lookout/style/typos/generation.py

```diff
@@ -198,41 +197,44 @@ def _lookup_corrections_for_token(self, typo_info: TypoInfo) -> List[Features]:
         candidates = []
         candidate_tokens = self._get_candidate_tokens(typo_info)
         typo_vec = self._vec(typo_info.typo)
-        dist_calc = EditDistance(typo_info.typo, "damerau")
-        for candidate in set(candidate_tokens):
-            candidate_vec = self.wv[candidate]
-            dist = dist_calc.damerau_levenshtein_distance(candidate, self.config["radius"])
+        for candidate, dist in set(candidate_tokens):
```

Comment:
`candidate_tokens` is set already 
```suggestion
        for candidate, dist in candidate_tokens:
```